controllers/doguInstallManager.go

```diff
@@ -166,13 +168,28 @@ func (m *doguInstallManager) Install(ctx context.Context, doguResource *k8sv1.Do
 		return fmt.Errorf("failed to pull image config: %w", err)
 	}
 
-	customK8sResources, err := m.fileExtractor.ExtractK8sResourcesFromContainer(ctx, doguResource, dogu)
+	m.recorder.Eventf(doguResource, corev1.EventTypeNormal, InstallEventReason, "Starting execPod...")
+	anExecPod, err := m.execPodFactory.NewExecPod(util.ExecPodVolumeModeInstall, doguResource, dogu)
+	if err != nil {
+		return fmt.Errorf("failed to create ExecPod resource %s: %w", anExecPod.ObjectKey().Name, err)
+	}
+	err = anExecPod.Create(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to create ExecPod %s: %w", anExecPod.ObjectKey().Name, err)
+	}
+	defer anExecPod.Delete(ctx)
```

Comment:
🚫 **[checkstyle]** <errcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `anExecPod.Delete` is not checked